src/features/sanctions/hooks/useChainalysisSanctioned.ts

```diff
@@ -0,0 +1,43 @@
+import { isAddress } from 'viem';
+import { useContractRead } from 'wagmi';
+
+import { useEvmAccount } from '../../wallet/hooks/evm';
+
+// https://go.chainalysis.com/chainalysis-oracle-docs.html
+const ORACLE_ABI = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'addr',
+        type: 'address',
+      },
+    ],
+    name: 'isSanctioned',
+    outputs: [
+      {
+        internalType: 'bool',
+        name: '',
+        type: 'bool',
+      },
+    ],
+    stateMutability: 'view',
+    type: 'function',
+  },
+] as const;
+const ORACLE_ADDRESS = '0x40C57923924B5c5c5455c48D93317139ADDaC8fb';
+
+export const useChainalysisSanctioned = () => {
```

Comment:
Nit: suggest `useIsAccountChainanalysisSanctioned`, my first thought was that this returned the list, not the account status